third_party/gtest-1.8.0/include/gtest/internal/gtest-param-util.h

```diff
@@ -57,9 +57,10 @@ namespace testing {
 // Consists of the parameter value and the integer parameter index.
 template <class ParamType>
 struct TestParamInfo {
-  TestParamInfo(const ParamType& a_param, size_t an_index) :
-    param(a_param),
-    index(an_index) {}
+  TestParamInfo(const ParamType& a_param, size_t an_index) {
```

Comment:
It doesn't work on zos with the following error:
"../../third_party/gtest-1.8.0/include/gtest/internal/gtest-param-util.h", line 62.19: CCN5861 (S) Too few arguments are specified for macro "index". Empty arguments are used.
"/usr/include/strings.h", line 206.16: CCN5425 (I) "index" is defined on line 206 of "/usr/include/strings.h".